dashboard/cron.py

```diff
@@ -118,28 +111,25 @@ def update_with_udw_access(request):
                         # query to retrieve all file access events for one course
                         query = 'select CAST(SUBSTR(JSON_EXTRACT_SCALAR(event, "$.object.id"), 35) AS STRING) AS FILE_ID, ' \
                                 'SUBSTR(JSON_EXTRACT_SCALAR(event, "$.membership.member.id"), 29) AS USER_ID, ' \
-                                'EVENT_TIME as ACCESS_TIME, ' \
-                                'SUBSTR(JSON_EXTRACT_SCALAR(event, "$.group.id"),31) AS COURSE_ID ' \
+                                'datetime(EVENT_TIME) as ACCESS_TIME ' \
                                 'FROM learning_datasets.enriched_events ' \
                                 'where JSON_EXTRACT_SCALAR(event, "$.edApp.id") = \'http://umich.instructure.com/\' ' \
                                 'and event_type = \'NavigationEvent\' ' \
                                 'and JSON_EXTRACT_SCALAR(event, "$.object.name") = \'attachment\' ' \
                                 'and JSON_EXTRACT_SCALAR(event, "$.action") = \'NavigatedTo\' ' \
                                 'and JSON_EXTRACT_SCALAR(event, "$.membership.member.id") is not null ' \
                                 'and SUBSTR(JSON_EXTRACT_SCALAR(event, "$.group.id"),31) = \'' + UDW_COURSE_ID + '\' '
-                        logger.debug(query)
+                        logger.info(query)
```

Comment:
I'd probably move these back to debug, was that what you were talking about?

Reply:
Yes. Changed it to debug.